SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +117,69 @@ etc...
+            },
+            {
+                "global_component_streaming_query": {
+                "component_name": "improbable.unreal.generated.yourprojectgamemode.YourProjectGameModeMultiClientRepData"
+                }
             }
-          }
-        ],
-        
+            ],
     
 ## Generating code for SpatialOS C++ workers and building your project
-1. Set your Unreal project to work with the SpatialOS Unreal GDK fork of the Unreal Engine, which you cloned and installed earlier. To do this:
-    1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
-    1. Right-click your uproject file and select **Switch Unreal Engine version**.
-    1. Select the path to the Unreal Engine fork you cloned earlier.
-    1. In your terminal, from `Plugins\<UnrealGDKRoot>\SpatialGDK\Build\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`
-    1. In File Explorer, right-click your uproject file and select **Generate Visual Studio project files**.
-    1. Open the solution in Visual Studio, and compile and run the project in the Development Editor configuration.
-
-## Running the Schema Generator and setting up the UE4 Editor
-1. In the Unreal Editor, on the SpatialOS Unreal GDK toolbar, click **Generate Schema** to run the `Schema Generator`.
-1. On the same toolbar, click **Snapshot**.
+Set your Unreal project to work with the SpatialOS GDK for Unreal fork of the Unreal Engine, which you cloned and installed earlier. To do this:
+1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
+2. Right-click your uproject file and select **Switch Unreal Engine version**.
+3. Select the path to the Unreal Engine fork you cloned earlier.
+4. In a terminal window, from `<ProjectRoot>\<GameRoot>\Plugins\UnrealGDK\SpatialGDK\Build\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`   
+For Example "`BuildWorker.bat StarterProjectEditor Win64 Development StarterProject.uproject`"  
+This builds and packages your workers ready for a Spatial deployment.
+5. In File Explorer, right-click your uproject file and select **Generate Visual Studio project files**.
+6. Open the solution in Visual Studio, and compile and run the project in the Development Editor configuration.
+
+## Running the Schema and Snapshot Generator
+1. In the Unreal Editor, on the SpatialOS Unreal GDK toolbar, click  the **Schema** button to run the `Schema Generator`.
+2. On the same toolbar, click the **Snapshot** button.
 
 ## Running your game
-1. In the Unreal Editor, on the toolbar, open the **Play** drop-down menu and check the box next to **Run Dedicated Server**.
-1. On the SpatialOS Unreal GDK toolbar, click **Launch** and wait for the output `SpatialOS ready`.
-1. On the main Unreal toolbar, click **Play**.
+1. In the Unreal Editor, on the toolbar, open the **Play** drop-down menu and check the box next to **Run Dedicated Server** as well as the checkbox for **Spatial Networking**
```

Comment:
Can you add screenshots here? This is arguably our most critical document.

Reply:
We should modify the engine to default dedicated server to on. It's an easy step to miss.

Reply:
Yeah let's include screenshots if possible.